src/index.css

```diff
@@ -24,10 +24,10 @@ a:hover {
 
 body {
   margin: 0;
-  display: flex;
+  /* display: flex;
   place-items: center;
   min-width: 320px;
-  min-height: 100vh;
+  min-height: 100vh; */
```

Comment:
make sure you delete comments / unused code when you commit

Reply:
Okey, thanks